pyproject.toml

```diff
@@ -54,6 +54,7 @@ sphinxemoji = {version="^0.1.8", optional=true}
 sphinx-copybutton = {version="^0.4.0", optional=true}
 rstcheck = {version="^3.3.1", optional=true}
 types-Pillow = {version="^8.3.3", optional=true}
+azure-mgmt-privatedns = "^1.0.0"
```

Comment:
1. make it optional like other azure packages.
2. Add it in the internal repo to make sure once it's merged, it won't break CI runs.

Reply:
done

Reply:
didn't see internal PR created.